### PR TITLE
refactor: excel cleanup

### PR DESCRIPTION
**Pull request by cpvalente**
cleans up the process for importing data from spreadsheets

It removes all imports for project data

@alex-Arc , I think I didnt break anything with the google sheet imports, but can you sanity check?

### CODE REVIEW

File: apps/client/src/features/modals/sheets-modal/SheetsModal.tsx [hunk 1]
```diff
@@ -84,12 +82,11 @@ export default function SheetsModal(props: SheetsModalProps) {
 
   const handleClose = () => {
     setRundown(null);
-    setProject(null);
     setUserFields(null);
     onClose();
   };
 
-  //SETP-1 Upload Client ID
+  //STEP-1 Upload Client ID
```
Comment (alex-Arc) [inline apps/client/src/features/modals/sheets-modal/SheetsModal.tsx:89]: sorry for al the spelling mistakes

File: apps/server/src/utils/parser.ts [hunk 2]
```diff
@@ -494,15 +422,16 @@ export const fileHandler = async (file: string, options: ExcelImportOptions): Pr
...
     res.data.userFields = parseUserFields(dataFromExcel);
 
-    await deleteFile(file);
+    deleteFile(file);
 
     return res;
   }
 
   if (file.endsWith('.json')) {
+    console.log('JSON!');
```
Comment (alex-Arc) [inline apps/server/src/utils/parser.ts:433]: console log :)

File: apps/server/src/utils/parser.ts [hunk 3]
```diff
@@ -512,4 +441,5 @@ export const fileHandler = async (file: string, options: ExcelImportOptions): Pr
     await configService.updateDatabaseConfig(fileName);
     return res;
   }
+  console.log('NOTHIGN');
```
Comment (alex-Arc) [inline apps/server/src/utils/parser.ts:444]: console log :)

**Review by alex-Arc — APPROVED** [review]
tested it an it works both up and down
and ignores project data

File: apps/server/src/utils/parser.ts (hunk 2 above)
Comment (cpvalente) [reply apps/server/src/utils/parser.ts:433]: good catch, this is fixed on the stacked PR

**Pull request merged**